invenio_rdm_migrator/state.py

```diff
@@ -196,6 +206,19 @@ def _initialize_db(cls, metadata):
             sqlite_autoincrement=False,
         )
 
+        sa.Table(
```

Comment:
add PIDs cache to be able to link PID creation to PID update (object_uuid, after the related draft has been created).